### PR TITLE
More players

### PR DESCRIPTION
**Pull request by Vic-ST**
Here are some new changes. @GoRedbirds These are updates for 2-4 players.

---
This change is [Reviewable](https://reviewable.io/reviews/stevencopeland/stores/45)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/StevenCopeland/Stores/pull/45" target="_blank">ReviewNinja

**Comment by Vic-ST** [comment]
Great!

---

Review status: 0 of 7 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/45#-:-KgLRH7WLWMNf3DZvdQ-:btojbfd)*

**Comment by Vic-ST** [comment]
Reviewed 7 of 7 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/45)*

File: JavaScript-Files/begin_round1.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
+    var beginRoundWith2 = function() {
```
Comment (Vic-ST) [inline JavaScript-Files/begin_round1.js:1]: I rename a lot of the functions !completed

**Comment by Vic-ST** [comment]
*[JavaScript-Files/begin_round1.js, line 1 at r1](https://reviewable.io:443/reviews/stevencopeland/stores/45#-KgLRuyIV-A_GcsAzT-G-r1-1:-KgMEYZJ4IV_-mrJ7q5p:bmfxrtz) ([raw file](https://github.com/stevencopeland/stores/blob/dfcd6f00b01375d83765fcbbc58f264068d89a8b/JavaScript-Files/begin_round1.js#L1)):*
<details><summary><i>Previously, StevenCopeland (Steven Copeland) wrote…</i></summary><blockquote>

I rename a lot of the functions !completed
</blockquote></details>

Done.hh

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/45)*

File: JavaScript-Files/begin_round1.js [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+        getMoneyM(ai3);
+        
+        var taxPayer = function() {
+            round += 1;
+            var roundsLeftUntilTaxDay = 6 - round;
+            alert("The round is round " + round + " which means you have " + roundsLeftUntilTaxDay + " days until taxday.");
+            if(round === 6) {
+                alert("Tax day: Everybody loses 10% of their money.");
+                //newPerson.money *= 0.90;
+                //person.money *= 0.90;
```
Comment (Vic-ST) [inline JavaScript-Files/begin_round1.js:142]: As I went through them I commented some out and un commented some.

**Review by Vic-ST — COMMENTED** [review]
This is pretty cool. I think I will work for the Stores game

**Comment by Vic-ST** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/45)*

**Comment by Vic-ST** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 12 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/45)*

**Comment by Vic-ST** [comment]
[CodeFactor](https://www.codefactor.io/repository/github/stevencopeland/stores) found multiple issues:

#### Duplicate Code
[JavaScript-Files\begin_round1.js:102-128
](https://github.com/StevenCopeland/Stores/blob/723d0b5c1da2025f47baf0a13a401840a2327ccd/JavaScript-Files/begin_round1.js#L102-L128)[JavaScript-Files\begin_round2.js:102-128
](https://github.com/StevenCopeland/Stores/blob/723d0b5c1da2025f47baf0a13a401840a2327ccd/JavaScript-Files/begin_round2.js#L102-L128)Commit 723d0b5c1da2025f47baf0a13a401840a2327ccd
[JavaScript-Files\begin_round2.js:103-124
](https://github.com/StevenCopeland/Stores/blob/58168b5eb9eb32fecddb84e8011fec97309b9713/JavaScript-Files/begin_round2.js#L103-L124)[JavaScript-Files\begin_round1.js:103-124
](https://github.com/StevenCopeland/Stores/blob/58168b5eb9eb32fecddb84e8011fec97309b9713/JavaScript-Files/begin_round1.js#L103-L124)[JavaScript-Files\begin_round3.js:103-124
](https://github.com/StevenCopeland/Stores/blob/58168b5eb9eb32fecddb84e8011fec97309b9713/JavaScript-Files/begin_round3.js#L103-L124)Commit 58168b5eb9eb32fecddb84e8011fec97309b9713
[JavaScript-Files\begin_round3.js:130-140
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round3.js#L130-L140)[JavaScript-Files\begin_round2.js:130-140
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round2.js#L130-L140)[JavaScript-Files\begin_round1.js:130-140
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round1.js#L130-L140)Commit 6b3b4292189ea9183f7a3a90d0b623153d024e72
[JavaScript-Files\begin_round2.js:161-188
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round2.js#L161-L188)[JavaScript-Files\begin_round3.js:161-188
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round3.js#L161-L188)[JavaScript-Files\begin_round1.js:161-188
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round1.js#L161-L188)Commit 6b3b4292189ea9183f7a3a90d0b623153d024e72
[JavaScript-Files\begin_round3.js:114-124
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round3.js#L114-L124)[JavaScript-Files\begin_round2.js:114-124
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round2.js#L114-L124)[JavaScript-Files\begin_round1.js:112-124
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round1.js#L112-L124)Commit 6b3b4292189ea9183f7a3a90d0b623153d024e72
[JavaScript-Files\begin_round1.js:143-156
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round1.js#L143-L156)[JavaScript-Files\begin_round3.js:143-156
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round3.js#L143-L156)[JavaScript-Files\begin_round2.js:143-156
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round2.js#L143-L156)Commit 6b3b4292189ea9183f7a3a90d0b623153d024e72
[JavaScript-Files\begin_round2.js:99-116
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round2.js#L99-L116)[JavaScript-Files\begin_round3.js:99-116
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round3.js#L99-L116)[JavaScript-Files\begin_round1.js:99-116
](https://github.com/StevenCopeland/Stores/blob/6b3b4292189ea9183f7a3a90d0b623153d024e72/JavaScript-Files/begin_round1.js#L99-L116)Commit 6b3b4292189ea9183f7a3a90d0b623153d024e72

**Comment by Vic-ST** [comment]
*[JavaScript-Files/howManyPlayers.js, line 1 at r4](https://reviewable.io:443/reviews/stevencopeland/stores/45#-KgLRuyFiD1Der8XiL23-r4-1:-KgU8wHylcSASCl50Qpy:btojbfd) ([raw file](https://github.com/stevencopeland/stores/blob/9852995b63c4beb585e875b9abd150ae48f8bd57/JavaScript-Files/howManyPlayers.js#L1)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: [Function 'anonymous' has a complexity of 22. (complexity)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=581796)
</blockquote></details>

Great!

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/45)*

**Comment by Vic-ST** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 8 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/45)*

**Comment by Vic-ST** [comment]
Review status: all files reviewed at latest revision, 8 unresolved discussions, some commit checks failed.

---

*[JavaScript-Files/howManyPlayers.js, line 4 at r4](https://reviewable.io:443/reviews/stevencopeland/stores/45#-KgLRuyFiD1Der8XiL23-r4-4:-KgU949Gjtowi8XMs2xe:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/9852995b63c4beb585e875b9abd150ae48f8bd57/JavaScript-Files/howManyPlayers.js#L4)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: ['me' is not defined. (no-undef)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=581796)
</blockquote></details>

Done.

---

*[JavaScript-Files/howManyPlayers.js, line 5 at r4](https://reviewable.io:443/reviews/stevencopeland/stores/45#-KgLRuyFiD1Der8XiL23-r4-5:-KgU958Cv2ODyBniV7Ut:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/9852995b63c4beb585e875b9abd150ae48f8bd57/JavaScript-Files/howManyPlayers.js#L5)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: ['beginRoundWith2' is not defined. (no-undef)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=581796)
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/45)*

**Comment by Vic-ST** [comment]
Review status: all files reviewed at latest revision, 8 unresolved discussions, some commit checks failed.

---

*[JavaScript-Files/howManyPlayers.js, line 8 at r4](https://reviewable.io:443/reviews/stevencopeland/stores/45#-KgLRuyFiD1Der8XiL23-r4-8:-KgU97x1PI0PwZGATlDd:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/9852995b63c4beb585e875b9abd150ae48f8bd57/JavaScript-Files/howManyPlayers.js#L8)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: ['me' is not defined. (no-undef)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=581796)
</blockquote></details>

Done.

---

*[JavaScript-Files/howManyPlayers.js, line 9 at r4](https://reviewable.io:443/reviews/stevencopeland/stores/45#-KgLRuyFiD1Der8XiL23-r4-9:-KgU98HOTbO1jCQtn8Lf:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/9852995b63c4beb585e875b9abd150ae48f8bd57/JavaScript-Files/howManyPlayers.js#L9)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: ['beginRoundWith3' is not defined. (no-undef)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=581796)
</blockquote></details>

Done.

---

*[JavaScript-Files/howManyPlayers.js, line 12 at r4](https://reviewable.io:443/reviews/stevencopeland/stores/45#-KgLRuyFiD1Der8XiL23-r4-12:-KgU98rIfIGFrHumR19z:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/9852995b63c4beb585e875b9abd150ae48f8bd57/JavaScript-Files/howManyPlayers.js#L12)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: ['me' is not defined. (no-undef)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=581796)
</blockquote></details>

Done.

---

*[JavaScript-Files/howManyPlayers.js, line 13 at r4](https://reviewable.io:443/reviews/stevencopeland/stores/45#-KgLRuyFiD1Der8XiL23-r4-13:-KgU9965HknpufBlpX9h:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/9852995b63c4beb585e875b9abd150ae48f8bd57/JavaScript-Files/howManyPlayers.js#L13)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: ['beginRoundWith4' is not defined. (no-undef)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=581796)
</blockquote></details>

Done.

---

*[JavaScript-Files/main.js, line 15 at r4](https://reviewable.io:443/reviews/stevencopeland/stores/45#-KgLQpATENExXaQ3NfhY-r4-15:-KgU99rEUlXpuOu7SorW:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/9852995b63c4beb585e875b9abd150ae48f8bd57/JavaScript-Files/main.js#L15)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: [A constructor name should not start with a lowercase letter. (new-cap)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=581796)
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/45)*

**Review by Vic-ST — COMMENTED** [review]
It looks good but since I have another branch I might just close this without merging.

**Comment by Vic-ST** [comment]
I had another branch for this. Closing now.

**Pull request closed without merging**